composer.json

```diff
@@ -15,7 +15,8 @@
 		}
 	],
 	"require": {
-		"php": ">=7.0"
+		"php": ">=7.0",
+		"ext-intl": "*"
```

Comment:
should probably remain only as suggested

Reply:
What should `Strings` class do when there is no intl extension?

Reply:
@hrach could you please elaborate on it more then just thumbs down? I think I know some of your arguments, but want to be sure what is on your mind.

Reply:
Well, I consider the Strings class as the main functionality of the nette/utils package and therefore it's dependencies should be required. Nothing more, nothing less.

Reply:
@hrach That's a valid opinion, but it's outside of scope of this PR. If you want to move currently suggested stuff to `require` you need another PR and do it consistently for all stuff in `suggest`.

Reply:
That makes sense, as that was my next question - what to do with the other dependencies.

Reply:
fixed in bb3a69cac366baa5ec7a5875099e12a12b48f9e2

Reply:
openned #151